layouts/default.vue

```diff
@@ -26,12 +26,17 @@
         </div>
 
 
-        <div class="ml-auto text-right text-sm my-auto">
+        <div class="ml-auto text-right text-sm d-none float-right md:d-inline-block">
```

Comment:
The right idea just not using flexbox! Changing to a display type of flex instead of inline-block would solve this problem with some modification to the flex scss file to remove the forced children selectors.